### ISSUE TITLE
Fails to download videos ("failed to find xoct event link")

### ISSUE THREAD

**Issue by FliegendeWurst** [FliegendeWurst/KIT-ILIAS-downloader#54]
The error message looks like this:

```
Syncing ..../ILIAS/6232703 – Straßenverkehrstechnik/Straßenverkehrstechnik WS 2021-2022: failed to process URL

Caused by:
    failed to find xoct event link
```

**Comment by probstlukas** [comment]
Still an issue:

```
Syncing ...../Robotik I/Robotik I - Einführung in die Robotik (WS242424152): failed to process URL

Caused by:
    failed to find xoct event link
```

Therefore, recordings have to be downloaded manually.

**Comment by FliegendeWurst** [comment]
Fixed in 0.3.8

**Comment by FliegendeWurst** [comment]
Hmm.. some videos still fail
```
Syncing 2021-04-22 02_Grundlagen_Modellierung_Teil_1.mp4: failed to process URL

Caused by:
    xoct player json not found
```

**Comment by FliegendeWurst** [comment]
Fixed in 0.3.9
